lib/assets/stylesheets/chef/components/logo-font.scss

```diff
@@ -0,0 +1,25 @@
+@font-face {
+  font-family: 'chef';
+  src: url('../fonts/chef.eot?-8vpdz');
```

Comment:
You'll want to use `font-url('chef.eot...')` for all of these `url`s. See http://compass-style.org/reference/compass/helpers/urls/#font-url.